src/errors.js

```diff
@@ -35,4 +36,21 @@ const exceptions = _.reduce(
   {}
 );
 
-module.exports = exceptions;
+const isRequireError = ({ name, message }) =>
+  name === 'ReferenceError' && message === 'require is not defined';
+
```

Comment:
I think this function is ok here for now. If we add more to it, then we can break it out to have more generic error handling.